lib/fs-driver.js

```diff
@@ -371,3 +352,14 @@ FsDriver.prototype.removeWorkingDirectory = function (cb) {
 FsDriver.prototype.workingDiff = function (cb) {
   this.diff(this.root, this.working, cb);
 };
+
+/**
+ * Exposes the escape method.
+ */
+Object.defineProperty(FsDriver, 'escape', {
+  value: function (string) {
+    return new FsDriver('/').escape(string);
+  },
+  writable: false,
+  enumerable: true
+});
```

Comment:
> roughly

yup - but in this case when you want to escape something you have to create a new `FsDriver` every time. that's what makes me go :/